packages/native-extensions/src/index.ts

```diff
@@ -0,0 +1,2 @@
+// Just so the package can be imported and be detected correctly by typescript
```

Comment:
When do we need to import this package in some TS file?

Reply:
I use the package to resolve the binding location, it's the safest way I found and it work while packaged, when running botpress or launching tests